readme.md

```diff
@@ -595,6 +595,74 @@ vrf keepalive
 
 ```
 
+#### Generate Switch Config With Overrides
+This option allows you to pass in a file that contains switch configuration that CANU will ignore on config generation.  A use case would be to ignore the site connection on spine01, or an edge device that CANU does not recognize.
+
+The override file type is yaml and a single file can be used for multiple switches.  You will need to specify the switch name and what config to ignore.
+
```

Comment:
Suggest a sentence here to say that overrides uses hier_config library and it's format.  Might want to link to the hier_config docs for this file.